apps/condo/domains/user/components/UserMenu.tsx

```diff
@@ -84,10 +84,10 @@ export const UserMenu = () => {
                 <StyledMenuItem onClick={goToOrganization}>
                     <Space size={16}>
                         <EnvironmentFilled style={menuIconStyles}/>
-                        <>
+                        <div title={organization.name}>
```

Comment:
Maybe, you can use Antd tooltips, as we are using them instead of the title attributes

https://ant.design/components/tooltip/

Reply:
It seems to me, that tooltips will bring too much of interactivity, that will feel annoying ;)
Asked design team about possibility to make width flexible for menu and for user string in header.

Reply:
As for me it's better to use components instead of div's

Reply:
So, latest news from design — remove organisation menu item
The menu should have only one item — "logout".